renderer/types/Minor.ts

```diff
@@ -1,11 +1,11 @@
 export type MinorParser = {
-    name: string;
-    help: string;
-    type: string;
-    metrics: {
-        value: string;
-        labels: {
-            [key: string]: string;
-        };
-    }[]
-}
\ No newline at end of file
+  name: string;
+  help: string;
+  type: string;
+  metrics: {
+    value: string;
+    labels: {
+      [key: string]: string;
+    };
+  }[];
+};
```

Comment:
we should re-use the types between main and rendered, because this is exactly where the types will be of most use, maybe make a types folder for type only files in the root of the repo?